KubeArmor/go.mod

```diff
@@ -19,42 +19,40 @@ replace (
 	github.com/kubearmor/KubeArmor/deployments => ../deployments
 	github.com/kubearmor/KubeArmor/pkg/KubeArmorController => ../pkg/KubeArmorController
 	github.com/kubearmor/KubeArmor/protobuf => ../protobuf
-	k8s.io/api => k8s.io/api v0.26.4
-	k8s.io/apimachinery => k8s.io/apimachinery v0.26.4
-	k8s.io/client-go => k8s.io/client-go v0.26.4
```

Comment:
I think only a `go mod tidy` won't update it unless someone runs `go get -u`.
But yes, you're right. We should have a replace directive otherwise once we integrate renovate, it will also try to update these to v0.27. :heavy_check_mark: